packages/cerebras/src/cerebras-provider.ts

```diff
@@ -87,6 +91,7 @@ export function createCerebras(
       headers: getHeaders,
       fetch: options.fetch,
       errorStructure: cerebrasErrorStructure,
+      supportsStructuredOutputs: options.supportsStructuredOutputs ?? false,
```

Comment:
@Zefty we enabled structured output by default now, without utilizing an option. Can you confirm if that works for you

https://github.com/vercel/ai/blob/8d0b9a0a1974446614fad5256fd8d8494051052e/packages/cerebras/src/cerebras-provider.ts#L96